src/jdk.jdwp.agent/windows/native/libjdwp/linker_md.c

```diff
@@ -108,13 +116,13 @@ dbgsysBuildLibName(char *holder, int holderlen, const char *pname, const char *f
     const int pnamelen = pname ? (int)strlen(pname) : 0;
 
     *holder = '\0';
```

Comment:
This assignment is now redundant: please remove it.

Reply:
Done.